core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -5977,6 +5977,66 @@ secret_configs:
     - build_farm
     name: cluster-init
     namespace: ci
+- from:
```

Comment:
This is unnecessary. Nothing will ever use that secret. @danilo-gemoli can you confirm since you are working on migration the whole secret flow?